.gitignore

```diff
@@ -1,5 +1,8 @@
-
 ### ide's
 ide
+
 # VS
-.vscode
\ No newline at end of file
+.vscode
+
+# git
```

Comment:
What's the point of this guy?

Reply:
These files are created by git during merges -- they are copies of an original file before merge for backup purposes. Documentation is [here](https://git-scm.com/docs/git-mergetool) see the "Temporary Files" section:

> `git mergetool` creates `*.orig` backup files while resolving merges. These are safe to remove once a file has been merged and its `git mergetool` session has completed

Reply:
Ok. Feel free to add to your local .gitignore, but I think we want to leave this out of the main .gitignore.

Reply:
I've reverted this.